pinot-core/src/main/java/org/apache/pinot/core/operator/transform/function/LookupTransformFunction.java

```diff
@@ -219,8 +218,7 @@ private void lookup(ValueBlock valueBlock, ValueAcceptor valueAcceptor) {
         }
       }
       // lookup
-      GenericRow row = _dataManager.lookupRowByPrimaryKey(primaryKey);
-      Object value = row == null ? null : row.getValue(_dimColumnName);
+      Object value = _dataManager.lookupValue(primaryKey, _dimColumnName);
```

Comment:
nice

but where lookupValues() method is used? or added for future use

Reply:
Sharp eyes! Yes it is a preparation for #13966